app/controllers/story.controller.js

```diff
@@ -384,3 +385,23 @@ exports.isFavorite = async (req, res) => {
         });
     }
 }
+
+
+// Find all Reviews for a story
+
+exports.findAllReviews = async (req, res) => {
+    
+    const storyId = req.params.storyId;
+    console.log(storyId);
+
+    try {
+        const data = await Review.findAll({ where: { storyId: storyId }, include: [{ model: User, as: "user", attributes: ["id", "firstname", "lastname"]}] });
+        res.send(data);
+    } catch (err) {
+        console.log(err);
```

Comment:
it's good to also log as much contextual information as possible, e.g.,
`console.log(``When fetching reviews for story ${storyId} encountered error ${err}``);`